concordia/tests/test_views.py

```diff
@@ -429,6 +434,121 @@ def test_asset_reservation_expiration(self):
         reservation = AssetTranscriptionReservation.objects.get()
         self.assertEqual(reservation.reservation_token, data["reservation_token"])
 
+    def test_asset_reservation_tombstone(self):
+        """
+        Simulate a tombstoned reservation which should:
+            - return 408 during the tombstone period
+            - during the tombstone period, another user may
+              obtain the reservation but the original user may not
+        """
+        asset = create_asset()
+        self.login_user()
+
+        reservation_token = token_hex(25)
```

Comment:
Can this block be replaced with a [`concordia.utils.get_or_create_reservation_token()`](https://github.com/LibraryOfCongress/concordia/blob/620632f1b984c225594c01d9f36deaef02374cac/concordia/utils.py#L27-L30) call? I'm a little itchy about having those drift out of sync.